cloudbuild-k8s.yaml

```diff
@@ -20,35 +25,86 @@ substitutions:
   _CLUSTER_LOCATION: us-central1
```

Comment:
Can you also add:
```yaml
_EXTRA_VARS: '{}'
_SOLUTION: null
```

Reply:
We should not include a default value for _SOLUTION_NAME.
It would be useful to include the default value for _EXTRA_VARS, so it becomes an optional argument for the triggers.